test/pytest/test_token_authorization.py

```diff
@@ -215,3 +193,25 @@ def test_token_expiration_time(setup_torchserve_expiration):
 
     response = requests.get("http://localhost:8081/models/mnist", headers=header)
     assert response.status_code == 400, "Token check failed"
+
+
+# Test priority between config.properties and cmd
+# config sets token authorization to true and cmd sets to false
+# Priority falls to cmd
+def test_priority():
+    MODEL_STORE = os.path.join(ROOT_DIR, "model_store/")
+    PLUGIN_STORE = os.path.join(ROOT_DIR, "plugins-path")
+
+    Path(test_utils.MODEL_STORE).mkdir(parents=True, exist_ok=True)
+    config_file_priority = os.path.join(
+        REPO_ROOT, "../resources/config_token_priority.properties"
+    )
+    test_utils.start_torchserve(
+        snapshot_file=config_file_priority, no_config_snapshots=True
+    )
+
+    response = requests.get(f"http://localhost:8081/models")
+
+    test_utils.stop_torchserve()
+
+    assert response.status_code == 200, "Token check failed"
```

Comment:
These is a test missing for the env variable and the different priorities with it.